src/components/graph/Graph.jsx

```diff
@@ -543,7 +550,8 @@ export default class Graph extends React.Component {
     // in order to properly update graph data we need to pause eventual d3 ongoing animations
     newGraphElements && this.pauseSimulation();
 
-    const transform = newConfig.panAndZoom !== this.state.config.panAndZoom ? 1 : this.state.transform;
+    const transform =
+      newConfig.panAndZoom !== this.state.config.panAndZoom ? { x: 0, y: 0, k: 1 } : this.state.transform;
```

Comment:
Having the line break right after the `=` looks a bit weird to me. 🤔 What about breaking the line in the ternary operator instead? I think something like this would look better:

```
const transform = newConfig.panAndZoom !== this.state.config.panAndZoom 
  ? { x: 0, y: 0, k: 1 }
  : this.state.transform;
```

Reply:
I think prettier is taking care of this? 🤔

Reply:
Yes, prettier is choosing the formatting, I don't have any control over this.